src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -315,14 +359,78 @@ export class QuickBar extends LitElement {
               `ui.dialogs.quick-bar.commands.server_control.${action}`
             )
           ),
-          icon: "hass:server",
+          icon: DEFAULT_SERVER_ICON,
           action: () => this.hass.callService("homeassistant", action),
         },
         this.hass.localize("ui.dialogs.generic.ok")
       )
     );
   }
 
+  private _generateNavigationCommands(): QuickBarItem[] {
+    const panelItems = this._generateNavigationPanelCommands();
+    const sectionItems = this._generateNavigationConfigSectionCommands();
+
+    return this._withNavigationActions([...panelItems, ...sectionItems]);
+  }
+
+  private _generateNavigationPanelCommands(): Omit<
+    QuickBarNavigationItem,
+    "action"
+  >[] {
+    return Object.keys(this.hass.panels).map((panelKey) => {
+      const panel = this.hass.panels[panelKey];
+
+      return {
+        text: getPanelText(this.hass, panel),
+        icon: getPanelIcon(panel) || DEFAULT_NAVIGATION_ICON,
+        path: `/${panel.url_path}`,
+      };
+    });
+  }
+
+  private _generateNavigationConfigSectionCommands(): Partial<
+    QuickBarNavigationItem
+  >[] {
+    const items: NavigationInfo[] = [];
+
+    for (const sectionKey of Object.keys(configSections)) {
+      for (const page of configSections[sectionKey]) {
+        if (canShowPage(this.hass, page)) {
+          if (page.component) {
+            const info = this._getNavigationInfoFromConfig(page);
+
+            if (info) {
+              items.push(info);
+            }
+          }
+        }
+      }
+    }
+
+    return items;
+  }
+
+  private _getNavigationInfoFromConfig(
+    page: PageNavigation
+  ): NavigationInfo | undefined {
+    const shortCaption = this.hass.localize(
+      `ui.dialogs.quick-bar.commands.navigation.${page.component}`
+    );
+
+    if (page && page.translationKey && shortCaption) {
```

Comment:
Can `page` be undefined? In that case like 418 will throw

Reply:
Looks like it can't be `undefined` anymore. I'll remove it

Reply:
But `page.component` itself can be `undefined`. The current code would handle that fine, but I can clean up some things by wrapping in an `if (page.component)`.

Reply:
Done